### PR TITLE
Update DG - Sort Command

### PR DESCRIPTION
**Pull request by aexolate**
The diagram reflects the code in #146.

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -154,6 +154,30 @@ Classes used by multiple components are in the `seedu.addressbook.commons` packa
...
+By returning the `SortedList` as an `ObservableList<Person>`, the UI will be able to update the person cards displayed
+after the `find` or `sort` command has been used.
+
+To facilitate comparing between two `Person` objects, the attributes also implement `Comparable<Person>`. The 
+`PersonCompmarators` class contains the comparators that can be used to compare persons based on the attribute.
+
+The command only allows for the sorting to be performed on one of the Person's attribute. The `SortCommandParser` will
+extract the attribute, and create a `SortCommand` using the corresponding `Comparator<Person>` for that attribute.
+
+#### Why it is implemented this way
```
Comment (Weiennn) [inline docs/DeveloperGuide.md:175]: Could we use a header here so it is more clear?
e.g. 
#### Design considerations:
**Aspect: using static comparators compared to lambda functions**

**Review by Weiennn — APPROVED** [review]
other than that LGTM!

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -154,6 +154,26 @@ Classes used by multiple components are in the `seedu.addressbook.commons` packa
...
+after the `find` or `sort` command has been used.
+
+To facilitate comparing between two `Person` objects, the attributes also implement `Comparable<Person>`. The
+comparators for the attributes can be used to then compare two Person on a specific attribute.
+
+The command only allows for the sorting to be performed on one of the Person's attribute. The `SortCommandParser` will
+extract the attribute, and create a `SortCommand` using the corresponding comparator for that attribute. The sequence 
+diagram below illustrates how the comparator is obtained and passed to the Model to be executed.
+
+<img src="images/SortSequenceDiagram.png" width="574" />
```
Comment (Goh-Li-Ting) [inline docs/DeveloperGuide.md:174]: Would you want to include the below?
```
<div markdown="span" class="alert alert-info">:information_source: **Note:** The lifeline for `sortCommandParser` should end at the destroy marker (X) but due to a limitation of PlantUML, the lifeline reaches the end of diagram.
```

**Review by Goh-Li-Ting — APPROVED** [review]
LGTM

**Pull request merged**